packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/configure.yml

```diff
@@ -11,6 +11,43 @@
 # grab local devices that should be blacklisted. This may expose
 # the local devices as FC storage in engine.
 
+- block:
+  - name: mask libvirt modular daemon units
+    systemd:
+      name: "{{ item }}"
+      state: stopped
+      enabled: no
+      masked: yes
+    with_items:
+      - virtqemud.service
+      - virtqemud.socket
+      - virtinterfaced.socket
+      - virtnetworkd.socket
+      - virtnodedevd.socket
+      - virtnwfilterd.socket
+      - virtsecretd.socket
+      - virtstoraged.socket
+      - virtproxyd.socket
+
+  - name: enable libvirtd monolithic daemon unit
+    systemd:
+      name: libvirtd
+      enabled: yes
+
+  - name: remove virtqemud socket
+    file:
+      path: /run/libvirt/virtqemud-sock
+      state: absent
```

Comment:
Why do we need to perform those steps when we are providing the preset? Shouldn't the preset force libvirt to disable split daemon functionality by itself during RPM installation? If not, what is the preset actually for when we need to perform everything manually?

Reply:
the issue of disabling the socket but the socket file doesn't remove is mentioned here [1], other than that the press does disable the libvirt daemons. 
[1] https://bugzilla.redhat.com/show_bug.cgi?id=802748

Reply:
Stopping/disabling socket units does not remove the socket files on the disk and the reason behind that is explained in the bug that @hbraha posted.

Libvirt client code uses probing to decide whether to use the modular daemons or the monolithic one. It first checks if the modular socket exists, and if it does, it assumes it should proceed with using modular daemons.

If we don't remove the socket file by ourselves we'll need to reboot hosts, which will make the socket file go away.

Reply:
OK, thanks for explanation